README.md

```diff
@@ -1,4 +1,56 @@
-# ActiveModelSerializers
+# ActiveModelSerializers (Deprecated)
+
+Please use [jsonapi-rb](https://github.com/jsonapi-rb) ([docs](http://jsonapi.org))
```

Comment:
ROAD and jsonapi-resources provide a layer on top of rails, whereas jsonapi-rb and ams are just (de)serialization